scrapy/resolver.py

```diff
@@ -16,8 +16,7 @@ def __init__(self, reactor, cache_size, timeout):
     def getHostByName(self, name, timeout=None):
         if name in dnscache:
             return defer.succeed(dnscache[name])
-        if not timeout:
-            timeout = self.timeout
+        timeout = (self.timeout,)
```

Comment:
np, it's me being obfuscated :)

the question is: Can `DNS_TIMEOUT` be at tuple?